src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -937,4 +937,20 @@
   <data name="ColorPicker_Editor.Text" xml:space="preserve">
     <value>Editor</value>
   </data>
+  <data name="PowerLauncher_Plugins.Text" xml:space="preserve">

```

Comment:
Let's not add more `PowerLauncher` prefix anywhere sine we will have to change it later on.
Use `PowerToysRun`.

Reply:
Updated branch